src/blog/truffle-v5-has-arrived.md

```diff
@@ -0,0 +1,102 @@
+We have released the most awaited version of Truffle just in time for the
+holidays! There are SO many exciting things to mention that will make your
+Truffle experience a rich one. Highlighted below are the new exciting things
+Truffle has to offer.
+
+[Download Truffle v5!](https://truffleframework.com/truffle)
+
+- **Bring your own compiler!!!!** You can now choose any solc-js version
+  available at solc-bin just tell Truffle which one you want and it will get
+  it. This includes the ability to use docker and native binary in an advanced
+  way.
+
+- **Web3 1.0** - the truffle-contract command is now using web3.js 1.0! This
+  will be great for error handling!
+
+- **New Migrations Command** - in v5 this command has been completely
+  rewritten. We have included better error messages, when a deployment fails
+  Truffle will tell you and give you an idea of possible fixes. Truffle will
+  now also give your more information about what is going on when you deploy a
+  contract, cost summaries and real time status updates about the amount of a
```

Comment:
I think you should use a serial comma here because one is used in a similar sentence below..."...your version
   number, the commands you run, and whether..."
So here would be "...contract, cost summaries, and real time..."